kuromoji-core/src/main/java/com/atilika/kuromoji/fst/FSTUtil.java

```diff
@@ -0,0 +1,68 @@
+package com.atilika.kuromoji.fst;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class FSTUtil {
+
+    List<String> inputStrings = new ArrayList<>();
+
+    /**
+     * Sort the input into Java-lexical order
+     *
+     * @param inputStream
+     * @throws IOException
+     */
+    public void sortInput(InputStream inputStream) throws IOException {
+
+        InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
```

Comment:
This code is trappy.  It's a good idea to do `InputStreamReader(inputStream, StandardCharsets.UTF-8)` instead to make sure encoding is always set as UTF-8 rather than relying on the platform's encoding.